src/components/common/OrderItemOverlay/OrderItemOverlay.module.scss

```diff
@@ -12,13 +17,20 @@
   right: 10px;
   top: 10px;
   font-size: 20px;
+  @include mobile-media {
+    display: none;
+  }
 }
 
 .header {
   display: flex;
   align-items: center;
   padding-right: 70px;
   padding-left: 55px;
+  @include mobile-media {
+    padding: $margin-small-4;
+    border-bottom: 1px solid rgba(363648, 0, 0, 0.1);
```

Comment:
You probably meant to do `border-bottom: 1px solid rgba(#363648, 0.1);` The 4 parameter version of rgba expects (red, green, blue, alpha), which means that the actual color for this will be a transparent red.

However, are those new mobile-only borders needed? As far as I can tell, the desktop styles already provide them and they seem to look like they should.